web/src/designer/components/Fields/BaseFieldEditor.tsx

```diff
@@ -170,15 +226,40 @@ export const BaseFieldEditor = ({fieldName, children}: Props) => {
       <Grid item xs={12}>
         <Card variant="outlined" sx={{p: 2}}>
           <Grid container spacing={2}>
-            <Grid item xs={12} md={6}>
+            <Grid item xs={12} md={4}>
               <DebouncedTextField
                 fullWidth
                 label="Label"
                 value={state.label}
                 onChange={e => updateProperty('label', e.target.value)}
               />
             </Grid>
-            <Grid item xs={12} md={6}>
+            <Grid item xs={12} md={4}>
+              <TextField
```

Comment:
This extra text field pushes the advanced help text widget into the right column making it too narrow. Maybe put the field id below the Label TextField?